guides/common/modules/proc_configuring-server-for-use-with-dhcpd.adoc

```diff
@@ -0,0 +1,107 @@
+[id="configuring-server-for-use-with-dhcpd"]
+= Configuring {ProjectServer} for use with dhcpd
+
+You can configure {ProductName} with a non-installer-managed DHCP server.
+
+Perform the steps on the {ProjectServer} or {SmartProxyServer}.
+
+
+.Prerequisites
+* xref:configuring-dhcpd-to-use-with-server[You configured the DHCP service and shared the configuration and lease files over the network].
+
+
+.Procedure
+
+. Install the required package:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {project-package-install} {nfs-client-package}
+----
+
+. Create the directories into which you later mount the NFS shares:
++
+[options="nowrap"]
+----
+# mkdir -p /mnt/nfs/etc/dhcp /mnt/nfs/var/lib/dhcpd
+----
+
+. Set the owner of the `/mnt/nfs` and sub-directories to `foreman-proxy`:
++
+[options="nowrap"]
+----
+# chown -R foreman-proxy /mnt/nfs
```

Comment:
I don't think this is needed, as long as `/mnt/nfs` is world readable (which is probably a safe bet). Though I'll also note that this whole provider really isn't that important.